cypress/integration/lock-exceptions.feature

```diff
@@ -8,12 +8,12 @@ Feature: Users should be able to manage their lock exceptions
 
   Scenario: User adds a lock exception
     When the user clicks on the 'Add lock exception' button
-    Then a modal appears
+    Then the user is sent to the add lock exception page
```

Comment:
The 'add lock exception' modal was extracted from the lock exceptions page and converted into its own separate page.